src/routes/settings/get.ts

```diff
@@ -11,6 +11,7 @@ interface Response {
   environment?: Environment;
   environmentRights?: string[];
   user: User;
+  notificationLevel: 0 | 1 | 2 | null;
```

Comment:
Can it be made optional without any dire consequences, and the Joi schema as well?